pkg/ingester/limiter.go

```diff
@@ -118,10 +121,13 @@ func (l *Limiter) FormatError(userID string, err error) error {
 func (l *Limiter) formatMaxSeriesPerUserError(userID string) error {
 	globalLimit := l.limits.MaxGlobalSeriesPerUser(userID)
 
-	return errors.New(globalerror.MaxSeriesPerUser.MessageWithPerTenantLimitConfig(
-		fmt.Sprintf("per-user series limit of %d exceeded", globalLimit),
-		validation.MaxSeriesPerUserFlag,
-	))
+	return log.SampledError{
+		Sampler: l.sampler,
```

Comment:
There's still just one sampler, but I'd be specific what this sampler is for, as I assume we don't want to reuse same sampler for different errors (because that would make it impossible to infer the amount of errors logged).

Reply:
My concern is that in the future someone would add a new error here and reuse the same sampler, because it's called "sampler" and not "perUserSeriesErrorSampler".

Anyway, this is totally unimportant, and definitely not blocking.

Reply:
I added another 6 errors.  As I understand your comment I am doing exactly what you didn't want.